sspl_test/alerts/node/test_node_bmc_interface.py

```diff
@@ -83,6 +72,23 @@ def test_bmc_interface(args):
     if bmc_interface_specific_info:
         assert(bmc_interface_specific_info.get("channel info") is not None)
 
+
+def test_bmc_interface(args):
+    check_sspl_ll_is_running()
+    # backup active bmc interface
+    BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL = backup_bmc_config()
+
+    if BMC_IF_CONSUL_VAL == "lan":
+        simulate_bmc_interface_alert.lan_channel_alert(BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL)
+    else:
+        simulate_bmc_interface_alert.kcs_channel_alert(BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL)
+    assert_triggers("fault", 25)
```

Comment:
why wait time of 25 & 35 ? it should match sspl polling frequency, which should be read from sspl config & then applied, default 30 otherwise.